src/controllers/commands/CommandController.cpp

```diff
@@ -434,6 +434,22 @@ void CommandController::initialize(Settings &, Paths &paths)
         userPopup->show();
         return "";
     });
+
+    this->registerCommand(
+        "/chatters", [](const auto & /*words*/, auto channel) {
+            auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+            if (twitchChannel == nullptr)
+            {
+                channel->addMessage(makeSystemMessage(
+                    "The /chatters command only works in Twitch Channels"));
+                return "";
+            }
+
+            channel->addMessage(makeSystemMessage(
+                "Chatter count: " +
+                QString::number(twitchChannel->chatterCount())));
```

Comment:
```suggestion
            const auto message =
                twitchChannel
                    ? "Chatter count: " +
                          QString::number(twitchChannel->chatterCount())
                    : "The /chatters command only works in Twitch Channels";
            channel->addMessage(makeSystemMessage(message));
```

Reply:
I don't agree that this is easier to read